src/locales/en/translation.json

```diff
@@ -230,11 +230,11 @@
       }
     },
     "pageIpvSuccess": {
-      "title": "You’ve successfully proved your identity",
-      "header": "You’ve successfully proved your identity",
+      "title": "Continue to the service you want to use",
+      "header": "Continue to the service you want to use",
```

Comment:
Might this change fail some of the UI tests?

Reply:
Hmm, you might be right – I'll check with the testers.

Reply:
The testing team is OK with merging in these changes, then updating their tests in the pipeline.